src/layouts/SettingsPage/SettingsPageDesktop.tsx

```diff
@@ -11,17 +10,23 @@ import { useScreenProps } from '../../hooks';
 export interface SettingsPageDesktopStyles {
 	title: TextStyle;
 	root: ViewStyle;
+	scrollViewContainer: ViewStyle;
 }
 
 export interface SettingsPageDesktopProps extends SettingsPageProps {
 	styles?: Partial<SettingsPageDesktopStyles>;
 }
 
+const { height } = Dimensions.get('window');
+
 const defaultStyles = (theme: Theme): SettingsPageDesktopStyles => ({
 	root: {
 		backgroundColor: theme.palette.background.default,
 		flex: 1,
 	},
+	scrollViewContainer: {
+		height: height - theme.spacing.unit * 10,
```

Comment:
use view height rather than window height

Reply:
if we use view height, then whole container starts scrolling(because view height is more than window height), that's why window height is required to stop scroll to go out of visible content window

Reply:
ok